.github/workflows/test-backend.yml

```
@@ -46,6 +47,22 @@ jobs:
 
       - run: docker compose --project-name $STACK_NAME down -v --remove-orphans
       - run: docker compose --project-name $STACK_NAME up -d db mailcatcher
+      
+      # Wait for database to be fully ready
+      - name: Wait for database
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:36: Double quote to prevent globbing and word splitting

(shellcheck)